lib/main.js

```diff
@@ -131,13 +131,15 @@ Lambda.prototype._params = function (program, buffer) {
       Variables: config
     }
   }
-  if (program.deadLetterConfigTargetArn !== undefined) {
+  if (program.deadLetterConfigTargetArn !== undefined && program.deadLetterConfigTargetArn !== '' && program.deadLetterConfigTargetArn !== null) {
     params.DeadLetterConfig = {
       TargetArn: program.deadLetterConfigTargetArn
     };
   }
-  if (program.tracingConfig) {
+  if (program.tracingConfig !== undefined && program.tracingConfig !== '' && program.tracingConfig !== null) {
     params.TracingConfig.Mode = program.tracingConfig;
+  } else {
+    delete params.TracingConfig;
```

Comment:
does this return an error if it's `null`? (seems counter-intuitive to the other params we're sending, but then again AWS API's are not known to be intuitive)